SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -706,7 +708,10 @@ bool SpatialGDKGenerateSchema()
 	GenerateSchemaFromClasses(TypeInfos, SchemaOutputPath, IdGenerator);
 	GenerateSchemaForSublevels(SchemaOutputPath, IdGenerator);
 	NextAvailableComponentId = IdGenerator.Peek();
-	SaveSchemaDatabase();
+	if (!SaveSchemaDatabase())
+	{
+		return false;
```

Comment:
How is this handled on the calling level? Is there a sensible error message? "Another process is locking this file" kinda stuff?

Reply:
There's an error message output to the log within the function when the actual call to access the file fails. One of my changes actually adjusted the wording of the message to something along the lines you suggest.

Reply:
👍👍👍👍👍👍👍👍👍